frontstage/controllers/party_controller.py

```diff
@@ -425,10 +428,8 @@ def get_survey_list_details_for_party(enrolment_data: dict, tag: str, business_p
     # Populate the cache with all case data
     caching_case_data(cache_data, business_ids, tag)
 
-    #  Populate the enrolments by creating a dictionary using the redis_cache
-    collection_exercises = {
-        survey_id: redis_cache.get_collection_exercises_by_survey(survey_id) for survey_id in surveys_ids
-    }
+    #  Populate the enrolments by creating a dict of enrolled collection exercises
+    collection_exercises = get_collection_exercises_for_surveys(surveys_ids, live_only=True)
```

Comment:
This line is the only change from the original PR
`collection_exercises = get_collection_exercises_for_surveys(surveys_ids)`
vs
`collection_exercises = get_collection_exercises_for_surveys(surveys_ids, live_only=True)`